app/src/main/res/layout/adapter_loading.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This layout is not used, I forgot to delete it. At first I added a ProgressBar at the end of the Upcoming Movies list, which used this layout, but it didn't work as plan, so I remove the feature.